gstprovider/gstthread.cpp

```diff
@@ -69,18 +69,18 @@ class CArgs
         count = args.count();
         if(count == 0)
         {
-            data = 0;
+            data = nullptr;
             argc = 0;
-            argv = 0;
+            argv = nullptr;
         }
         else
         {
-            data = (char **)malloc(sizeof(char **) * count);
-            argv = (char **)malloc(sizeof(char **) * count);
+            data = static_cast<char **>(malloc(sizeof(char **) * static_cast<unsigned long>(count)));
```

Comment:
in fact there is `sizeof(char **)`, but doesn't make difference anyway.
And quintptr(count) would look better than `static_cast<unsigned long>(count)`

Reply:
done